i8_terminal/common/layout.py

```diff
@@ -30,6 +39,12 @@ def df2Table(df: DataFrame, style_profile: str = "default", columns_justify: Dic
         "Revenue Cons.": "right",
         "Revenue Actual": "right",
         "Level": "right",
+        "Basic Earnings per Share": "right",
```

Comment:
I think we discussed that based on the display type of the metric, we can map the alignment. Just create a dictionary with all display types and map them to `right` or `left`